tests/sles4sap/hana.pm

```diff
@@ -0,0 +1,139 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Checks HANA Wizard installation on Gnome
```

Comment:
Could you please provide a little bit more details? Maybe even avoid words like "Test" or "Check" because that would be pretty obvious for tests :) E.g. what is the desired outcome that this module looks for?